packages/v4/src/content/design-guidelines/components/wizard/wizard.md

```diff
@@ -72,6 +72,21 @@ Sub-steps can be added to the sidebar. Use sub-steps when there is a hierarchica
 
 * Parent steps are automatically closed once they are complete. However, the user can manually expand the parent back open at any time to see the sub-steps again while staying in context of their current step.
 
+### Wizard with optional steps
+Optional steps can be added to the sidebar. Use optional steps when there are steps that are do not need to be filled out to complete the wizard.
+
+<img src="./img/wizard-with-optional-steps.png" alt="Wizard with optional steps" />
+
+
+1. **Optional steps:** Optional steps are always nested inside of a major step.
```

Comment:
Is this true? Could there not just be an optional step on it's own (without nested items)?? what if it's just one simple screen?